subprojects/plugins/src/integTest/groovy/org/gradle/java/JavaLibraryFeatureCompilationIntegrationTest.groovy

```diff
@@ -550,7 +550,7 @@ class JavaLibraryFeatureCompilationIntegrationTest extends AbstractIntegrationSp
                 [apiElements, runtimeElements].each {
                     assert it.canBeConsumed == true
                     assert it.canBeResolved == false
-                    assert it.canBeDeclaredAgainst == true
+                    assert it.canBeDeclaredAgainst == false
```

Comment:
This should not change. This was the behavior regression that I just fixed in 8.0 RC2